typescript/core/src/types.ts

```diff
@@ -129,7 +129,7 @@ type Values<T> = T[keyof T];
 export type TypedMcapRecord = Values<TypedMcapRecords>;
 export type McapRecord = Values<McapRecords>;
 
-export interface McapStreamReader {
+export interface IMcapStreamReader {
```

Comment:
I don't recall why we had this interface but I suspect we can remove it, since we are exporting the McapStreamReader class anyway...